src/commands/support/simbridge.ts

```diff
@@ -0,0 +1,24 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+export const simbridge: CommandDefinition = {
+    name: 'simbridge',
+    description: 'Describe SimBridge and provide support information',
+    category: CommandCategory.SUPPORT,
+    executor: async (msg) => {
+        const simbridgeEmbed = makeEmbed({
+            title: 'FlyByWire Support | SimBridge',
+            description: makeLines([
+                'SimBridge is our custom-built solution to connect our aircraft to various external devices and data. '
+                + 'It is found in the FBW Installer and ensures features will function seamlessly requiring fewer extra steps before launching into your flights.',
+                '',
+                '> **Important Note:** Please keep SimBridge updated at all times regardless of the version of aircraft you are currently flying! ',
+                '',
+                'Please [**read our documentation here**](https://docs.flybywiresim.com/simbridge) for additional information and status of associated features.',
+            ]),
+        });
+
+        await msg.channel.send({ embeds: [simbridgeEmbed] });
```

Comment:
**suggestion:** return instead of await so that error handling works better

Reply:
Added this as a separate commit in case of the need to revert but the rest of our commands use await // unless the bot-team is doing this as part of the refactor etc? Either way change made :)!

Reply:
Ideally they should all return it, looks like something that got lost in time as we made that change, albeit that was a LONG while ago

Reply:
example in ping:

https://github.com/flybywiresim/discord-bot/blob/b7aa533b02c50d8eb88c5ead2d55708d253e747c/src/commands/utils/ping.ts#L23

Reply:
We're gonna have a lot to refactor. Thanks for bringing this up